README.md

```diff
@@ -202,14 +214,14 @@ Please use the following [code style](https://github.com/airbnb/javascript) as m
 - `process.env.APP_TENANT_ID` - The GUID of the tenant (such as `alazurealertlogic.onmicrosoft.com`)
 - `process.evn.APP_RESOURCE_GROUP` - The name of the resource group where you deployed your application.
 - `process.env.CUSTOMCONNSTR_APP_CLIENT_ID` - The GUID of your application that created the subscription.
-**Note** You can obtain this value from _Azure_ -> _AD_ -> _App registrations_ -> _Your app name_
+**Note** You can obtain this value from _Azure_ > _AD_ > _App registrations_ > _Your app name_
 - `process.env.CUSTOMCONNSTR_APP_CLIENT_SECRET` - The secret key of your application from _App Registrations_.
 - `process.env.CUSTOMCONNSTR_APP_CI_ACCESS_KEY_ID` - The access key returned from AIMs [above](#create_an_alert_logic_access_key).
 - `process.env.CUSTOMCONNSTR_APP_CI_SECRET_KEY`- The secret key returned from AIMs [above](#create_an_alert_logic_access_key).
 
 
 # Known Issues/ Open Questions
-
+<!--Do we want to publish this in customer-facing documentation? If so, let's talk about making this statement more useful for customers.-->
```

Comment:
Probably not.  Could you please remove it.

Reply:
Removed full section, because removal of the one issue means the section has no content.